### PR TITLE
Resizable table columns

### PR DESCRIPTION
**Pull request by nathaniellee**
## PR Checklist

- Manually tested across supported browsers
  - [x] Chrome
  - [x] Firefox
  - [x] Safari
  - [x] IE11 (Win 7)
  - [x] Edge (Win 10)
- [x] Unit tests written (`common` at minimum)
- [x] PR has one of the `semver-` labels
- [x] Two core team engineer approvals
- [x] One core team UX approval

### CODE REVIEW

File: src/components/Table/Table.less [hunk 1]
```diff
@@ -67,6 +68,19 @@
...
+			&.lucid-Table-is-resizable {
+				line-height: inherit;
+				padding: 0;
+				&[rowspan='2'] {
+					.lucid-Table-is-resizable-content	{
+						line-height: 53px;
+					}
+				}
+				.lucid-Table-is-resizable-content {
+					padding: 0 @Table-cell-padding - 4 0 @Table-cell-padding;
```
Comment (jondlm) [inline src/components/Table/Table.less]: This could use some parenthesis for clarity

File: src/components/Table/Table.jsx [hunk 2]
```diff
@@ -152,6 +155,26 @@ const Tr = React.createClass(createLucidComponentDefinition({
 	}
 }));
 
+const ResizableThContent = ({ children, onDrag, onDragEnd, onDragStart }) => (
```
Comment (jondlm) [inline src/components/Table/Table.jsx]: Is this necessary since it's only used in one place?

File: src/components/Table/Table.jsx [hunk 3]
```diff
@@ -206,33 +261,102 @@ const Th = React.createClass(createLucidComponentDefinition({
...
+	getWidth() {
+		return this._root.getBoundingClientRect().width;
+	},
+	handleDragEnded(coordinates, { event }) {
+		this.setState({
+			isResizing: false,
+			passiveWidth: this.state.activeWidth
+		});
+
+		window.document.body.style.cursor = 'auto';
```
Comment (jondlm) [inline src/components/Table/Table.jsx]: Haha, nice ;)

File: src/components/Table/Table.jsx (hunk 2 above)
Comment (nathaniellee) [inline src/components/Table/Table.jsx]: No, it's not. Our original render used it a couple of times, but we ended up being able to refactor the logic to only utilize this one time. We missed the opportunity to eliminate the component in all of our euphoria. ;)

File: src/components/Table/Table.jsx [hunk 4]
```diff
@@ -4,10 +4,13 @@ import { lucidClassNames } from '../../util/style-helpers';
 import { createLucidComponentDefinition } from '../../util/component-definition';
 import { findElementsByType } from '../../util/child-component';
 import CaretIcon from '../Icon/CaretIcon/CaretIcon';
+import DragCaptureZone from '../DragCaptureZone/DragCaptureZone';
```
Comment (jondlm) [inline src/components/Table/Table.jsx]: Add `madeFrom`

Comment (jondlm) [inline src/components/Table/Table.jsx]: I went ahead and added it for you :)

**Pull request merged**